src/wx/guiinit.cpp

```diff
@@ -3188,6 +3188,7 @@ bool MainFrame::BindControls()
 
         d = LoadXRCDialog("UIConfig");
         { getcbb("HideMenuBar", gopts.hide_menu_bar); }
+        { getcbb("SuspendScreenSaver", gopts.suspend_screensaver); }
```

Comment:
We should hide the checkbox on Windows and mac until we have an implementation for those platforms. But you don't have to do this, I can do this after I merge your change.